### PR TITLE
remove an analysis server instance

### PR DESCRIPTION
**Pull request by devoncarew**
- remove an analysis server instance (instead of starting two analysis server instances, just start one)
- remove some analysis server warmup code
- remove older code related to v1 of the dart services api

This should reduce the memory usage of a dart services instance and will moderately improve instance startup time.

---

- [x] I’ve reviewed the contributor guide and applied the relevant portions to this PR.

<details>
  <summary>Contribution guidelines:</summary><br>

- See our [contributor guide](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md) for general expectations for PRs.
- Larger or significant changes should be discussed in an issue before creating a PR.
- Contributions to our repos should follow the [Dart style guide](https://dart.dev/guides/language/effective-dart) and use `dart format`.
- Most changes should add an entry to the changelog and may need to [rev the pubspec package version](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change).
- Changes to packages require [corresponding tests](https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md#Testing).

Note that many Dart repos have a weekly cadence for reviewing PRs - please allow for some latency before initial review feedback.
</details>

### CODE REVIEW

File: pkgs/dart_services/doc/generated/dartservices.dart
Comment (johnpryan) [inline pkgs/dart_services/doc/generated/dartservices.dart:1]: There's also a JSON file in this directory - is that file still required?

Comment (devoncarew) [reply pkgs/dart_services/doc/generated/dartservices.dart:1]: No - I can delete it; it was interesting to see what a JSON only interface to the services could look like. I'll put a link in this PR to the file for posterity.

**Comment by devoncarew** [comment]
https://github.com/dart-lang/dart-pad/blob/3bdbf533f5935b6dd4497781b56d8bed5569eaff/pkgs/dart_services/doc/generated/dartservices.json

and

https://github.com/dart-lang/dart-pad/blob/3bdbf533f5935b6dd4497781b56d8bed5569eaff/pkgs/dart_services/doc/generated/dartservices.dart
